src/test/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+my.twitter.baseurl=http://localhost:8092/2
+my.twitter.token=abc
+my.twitter.timeout=5000
+
+spring.datasource.url=jdbc:tc:postgresql:10.11:///warn_me
```

Comment:
I forgot to set postgres testcontainers to version from the project which is `13.3` you can change it